android-components/components/support/utils/src/test/java/mozilla/components/support/utils/BootUtilsTest.kt

```diff
@@ -1,20 +1,20 @@
-package org.mozilla.fenix.utils
+package mozilla.components.support.utils
 
 import android.os.Build
 import io.mockk.every
 import io.mockk.mockk
 import mozilla.components.support.test.robolectric.testContext
+import mozilla.components.support.utils.BootUtils.Companion.getBootIdentifier
 import org.junit.Assert.assertEquals
 import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith
-import org.mozilla.fenix.helpers.FenixRobolectricTestRunner
-import org.mozilla.fenix.utils.BootUtils.Companion.getBootIdentifier
+import org.robolectric.RobolectricTestRunner
 import org.robolectric.annotation.Config
 
 private const val NO_BOOT_IDENTIFIER = "no boot identifier available"
 
-@RunWith(FenixRobolectricTestRunner::class)
+@RunWith(RobolectricTestRunner::class)
```

Comment:
We don't need any special test runner here. The Junit default one would work and is always best to use when possible.